README.md

```diff
@@ -67,6 +67,11 @@ So if you want to get all the examples, make sure to clone the repo recursively
     <td>Python</td>
     <td>Generate 3D animation data of a single person from a single video from an unknown camera</td>
   </tr>
+  <tr>
+    <td><a href="examples/Move_API_single_person_multiple_video_go_pro.ipynb">Move_API_single_person_multiple_video_go_pro.ipynb</a></td>
+    <td>Python</td>
+    <td>Generate 3D animation data of a single person from multiple videos from a gopro</td>
```

Comment:
We support other cameras though - not just gopro?

Reply:
This is just for an example. The list of cameras are added in the readme [here](https://github.com/move-ai/recipes/pull/17)